app/commands/v2/publish.rb

```diff
@@ -171,19 +174,29 @@ def update_dependencies?
         EditionDiff.new(edition).field_diff.present?
       end
 
-      def send_downstream(content_id, locale, update_type)
-        return unless downstream
-
+      def send_downstream_live
         queue = update_type == 'republish' ? DownstreamLiveWorker::LOW_QUEUE : DownstreamLiveWorker::HIGH_QUEUE
-
         DownstreamLiveWorker.perform_async_in_queue(
           queue,
+          worker_params
+        )
+      end
+
+      def send_downstream_draft
+        queue = update_type == 'republish' ? DownstreamDraftWorker::LOW_QUEUE : DownstreamDraftWorker::HIGH_QUEUE
+        DownstreamDraftWorker.perform_async_in_queue(
+          queue,
+          worker_params.merge(update_dependencies: update_dependencies?)
+        )
+      end
+
+      def worker_params
+        {
           content_id: content_id,
           locale: locale,
           message_queue_update_type: update_type,
```

Comment:
It's not of major consequence but the downstream draft worker doesn't accept this attribute by the way.